### PR TITLE
Add my first domain

### PR DESCRIPTION
**Pull request by Daktyl198**
pull this~

### CODE REVIEW

File: private/daktyl198.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "selector": "^download\\.orinjido\\.hype$",
+            "records" : [
+                {
+                    "domain_name": "download.orinjido.hype",
+                    "rtype" : "AAAA",
+                    "rdata": "fcb4:971f:7e31:420:60bd:7fdc:b686:5493"
+                }
+            ],
+            "targets" : []
+        },
```
Comment (MaddieM4) [inline private/daktyl198.json]: Your PR wasn't working (SERVFAIL errors whenever I tried to resolve any orinjido.hype domains), so I actually improved DJDNS debugging to figure out why. Turns out it was just the JSON parser crashing from a trailing comma. You take that out, and it should just work, beautifully.

Let me know when this is ready for merge, I know there was some more stuff you were intending to do on your end.

Comment (Daktyl198) [inline private/daktyl198.json]: You know, I was JUST (and I mean JUST) looking at that comma and wondering if it was going to mess things up.

**Comment by MaddieM4** [comment]
k, is this ready to merge after I test this latest commit? Like, is all the data correct and everything?

**Comment by Daktyl198** [comment]
I'm pretty sure, yeah.

**Comment by MaddieM4** [comment]
Merged. Should be live on my public server shortly.

**Pull request merged**